wiki/Do_you_really_want_to_ask_peppy/pl.md

```diff
@@ -0,0 +1,15 @@
+# Czy naprawdę chcesz skontaktować się z peppym?
+
+Cześć. Ja ([peppy](/users/2)) rozumiem twoją chęć do prywatnego skontaktowania się ze mną, ale musisz najpierw zrozumieć, że to **nie** jest miejsce,  gdzie możesz mnie poprosić o pomoc. Oddzielam pomoc techniczną dla osu! od prywatnych wiadomości, a samo nadążanie za tymi drugimi jest już wystarczająco trudne i męczące. W związku z tym pragnę przyciągnąć twoją uwagę do poniższych opcji. Skorzystaj z tej, która najbardziej odpowiada twojemu problemowi.
+
+## Pomoc techniczna
+
+Większość problemów technicznych można rozwiązać poprzez znalezienie odpowiedniego wątku (lub założenie nowego) na [forum pomocy technicznej](/community/forums/5). Jest tam wiele innych osób, które mogą ci pomóc, więc znajdziesz rozwiązanie szybciej, niż kontaktując się ze mną.
+
+## Pomoc ogólna
+
+Dla problemów związanych z kontem, opłatami, korespondencji dotyczącej blokad konta, pytań odnośnie turniejów oraz **wszystkich pozostałych spraw** wyślij e-maila na adres [accounts@ppy.sh](mailto:accounts@ppy.sh) (pytania dotyczące kont) lub [osu@ppy.sh](mailto:osu@ppy.sh) (pozostałe). Jeśli któryś z członków zespołu pomocy technicznej jest dostępny, możesz użyć małego przycisku ze znakiem zapytania, który pojawi się w prawym dolnym rogu.
```

Comment:
wyślij maila
Usuń ostatnie zdanie

Reply:
Osobiście wolałbym „e-maila”, bo jest to pełniejsza forma, która bardziej pasuje do ogólnego stylu wiadomości.